README.md

```diff
@@ -123,6 +123,19 @@ tracking down why `static_ips()` calls fail.
 
 Check out the [static_ips() example](#static_ips)
 
+### But I Want To Make Strings!!
+
+Yeah, `spruce` can do that!
+
+```yml
+env: production
+cluster:
+  name: mjolnir
+ident: (( string cluster.name "//" env ))
```

Comment:
so far they've all been verbs - and it's a concatenation operator, so concat just seemed more intuitive